packages/design-system/src/types/index.ts

```diff
@@ -0,0 +1,5 @@
+export type DataAttributes = {
+	'data-feature'?: string | undefined;
+	'data-test'?: string | undefined;
+	'data-testid'?: string | undefined;
```

Comment:
```suggestion
	'data-feature'?: string;
	'data-test'?: string;
	'data-testid'?: string;
```
If one choses to use a data-attribute, they should not set it to undefined